iceoryx2-bb/cxx/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache Software License 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+# which is available at https://opensource.org/licenses/MIT.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+cmake_minimum_required(VERSION 3.22)
+
+project(iceoryx2-bb VERSION ${IOX2_VERSION_STRING} LANGUAGES CXX)
```

Comment:
Food for thought. Is there a chance that we offer some kind of C API for this container? In this case it would maybe make sense to use `iceoryx2-bb-cxx` or maybe `iceoryx2-cxx-bb`, which would have the nice side effect to have everything that is C++ related have the `iceoryx2-cxx` prefix